src/test/java/de/fom/webapp/integration/controller/CardSetControllerTest.java

```diff
@@ -138,4 +138,27 @@ void testSearchCardSetsWithTagsBogus() throws Exception{
                 .andExpect(content().contentType(APPLICATION_JSON))
                 .andExpect(jsonPath("$.content.[0].tags").doesNotExist());
     }
+    @Test
+    void testSelectCardSetById() throws Exception {
+        this.mockMvc.perform(
+                get("/api/selectCardSet")
+                        .param("cardSetId", "1")
+        )
+                .andDo(print())
+                .andExpect(status().isOk())
+                .andExpect(content().contentType(APPLICATION_JSON))
+                .andExpect(jsonPath("$.content.[0].id").value(1));
+    }
+
+    @Test
+    void testSelectCardSetByIdBogus() throws Exception {
+        this.mockMvc.perform(
+                get("/api/selectCardSet")
+                        .param("cardSetId","Herr der Ringe")
+        )
+                .andDo(print())
+                .andExpect(status().isOk())
```

Comment:
Wie oben beschrieben sollte es hier zu einem error kommen, also nicht OK(200) sonder Not Found(404)